tests/agent-plugin-unit/test_mk_jolokia.py

```diff
@@ -117,7 +117,7 @@ def test_jolokia_yield_configured_instances() -> None:
 class _MockHttpResponse(object):  # pylint: disable=useless-object-inheritance
```

Comment:
I suspect that these tests might be executed in a python2-only container during our tests, so I think they have to stay as well.

Reply:
Reverted.